### PR TITLE
Add New Topology t0-80

### PR DESCRIPTION
**Pull request by chitra-raghavan**
### Description of PR
Add new T0-80 topology which is based out of T0-64 topology

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### How did you verify/test it?
./testbed-cli.sh remove-topo
./testbed-cli.sh add-topo
./testbed-cli.sh deploy-mg

### CODE REVIEW

File: ansible/vars/topo_t0-80.yml [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+    ARISTA03T1:
+      vlans:
+        - 40
+        - 41
+      vm_offset: 2
+    ARISTA04T1:
+      vlans:
+        - 42
+        - 43
+      vm_offset: 3
```
Comment (daall) [inline ansible/vars/topo_t0-80.yml:97]: I would add VMs so that the 16-19 and 60-63 uplinks are also in use.

Comment (chitra-raghavan) [reply ansible/vars/topo_t0-80.yml:97]: sure Danny, Will include other uplink ports also.

Comment (yxieca) [reply ansible/vars/topo_t0-80.yml:97]: @chitra-raghavan did this PR fall out of your rader?

Comment (chitra-raghavan) [reply ansible/vars/topo_t0-80.yml:97]: @yxieca , working on this PR.

Comment (SubhajitPalKeysight) [reply ansible/vars/topo_t0-80.yml:97]: Would you please also include a documentation of updated topology ( https://github.com/Azure/sonic-mgmt/blob/master/docs/testbed/README.testbed.Topology.md )

**Comment by chitra-raghavan** [comment]
Updated the topology according to new port_config.ini as per  https://github.com/Azure/sonic-buildimage/pull/6906

File: ansible/vars/topo_t0-80.yml (hunk 1 above)
Comment (chitra-raghavan) [reply ansible/vars/topo_t0-80.yml]: Updated the README file

**Comment by chitra-raghavan** [comment]
show command for the topology is attached. 
[Topology_t0-80.txt](https://github.com/Azure/sonic-mgmt/files/6090480/Topology_t0-80.txt)

**Review by daall — APPROVED** [review]

File: ansible/vars/topo_t0-80.yml (hunk 1 above)
Comment (SubhajitPalKeysight) [reply ansible/vars/topo_t0-80.yml]: thanks

**Pull request merged**